docs/reference/free-functions.rst

```diff
@@ -59,6 +60,7 @@ Math
    round
    sqrt
    subtract
+   unary_minus
```

Comment:
Numpy calls this differently? https://numpy.org/doc/stable/reference/generated/numpy.negative.html

Reply:
That's why I didn't find it! Renaming...